.travis/build-site.sh

```diff
@@ -44,20 +32,7 @@ fi
 
 (
     if [[ "$VERSION" == *-SNAPSHOT && "$TRAVIS_BRANCH" == "master" ]]; then
-        # this can take very long and no output is generated. therefore use the background job again
-        export PING_SLEEP=30s
-        export BUILD_OUTPUT=/tmp/build-site-upload.out
-        export PING_PID_FILE=/tmp/build-site-upload-ping.pid
-
-        source .travis/background-job-funcs.sh
-
         # Uploading snapshot site...
-        rsync -ah --stats --delete target/pmd-doc-${VERSION}/ ${PMD_SF_USER}@web.sourceforge.net:/home/project-web/pmd/htdocs/snapshot/
-
-        # The build finished without returning an error so dump a tail of the output
-        dump_output
-
-        # nicely terminate the ping output loop
-        kill_ping
+        travis_wait rsync -ah --stats --delete target/pmd-doc-${VERSION}/ ${PMD_SF_USER}@web.sourceforge.net:/home/project-web/pmd/htdocs/snapshot/
```

Comment:
this one seems fixable... just replace `bash .travis/build-$BUILD.sh` with `./.travis/build-$BUILD.sh` (given proper execution permissions) or `source .travis/build-$BUILD.sh` to run on the same bash